src/krux/pages/home_pages/wallet_descriptor.py

```diff
@@ -177,7 +178,16 @@ def display_loading_wallet(self, wallet):
                 key_origin_str = str(key.origin)
                 key_fingerprint += key_origin_str[:8]
             else:
-                key_fingerprint += t("unknown")
+                if (
```

Comment:
Does this need a wallet-sans-key condition for reviewing descriptor w/o loading wallet first?  Else wallet.key is None and won't have policy_type or script_type attributes?

Asking because I tried loading Gui's TR descriptor via tools/wallet-descriptor... and saw "NoneType has no policy_type", but not sure if it's from this line or the elif at line 216

Reply:
Thank you!
Fixed by getting this information from descriptor instead of loaded key (the right way)